README.md

```diff
@@ -52,19 +52,19 @@ The global configurations can be initialized with ENV values.  The following con
   This is the Account ID that is used in the path to API endpoints that require an account ID.  This
```

Comment:
Readme should send gem consumer to yext documentation for coverage on available model attributes.